kusto/kusto.go

```diff
@@ -286,6 +280,17 @@
 			return nil, errors.ES(op, errors.KClientArgs, "QueryValues in the the Stmt were incorrect: %s", err).SetNoRetry()
 		}
 	}
+	if query.SupportsParameters() {
+		if len(opt.requestProperties.QueryParameters.ToParameterCollection()) != 0 {
+			return nil, errors.ES(op, errors.KClientArgs, "Cannot use both Stmt and StatementQueryParameters. Please use statementBuilder instead of Stmt.").SetNoRetry()
```

Comment:
"kusto.Stmt does not support the QueryParameters option. Construct your query using kql.Builder"